### PR TITLE
add support for oracle linux 8

### PR DESCRIPTION
**Pull request by tomerd**
motivation: with centos 8 deprecated, teams are moving to oracle linux 8

changes:
* add RPM spec and auxilary setup for building RPMs for oracle linux 8

### CODE REVIEW

**Comment by tomerd** [comment]
builds o top of #87 for sharing across RPM builders

**Comment by tomerd** [comment]
related: https://github.com/apple/swift-docker/pull/272

File: platforms/Linux/RPM/Oraclelinux/8/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Building Swift on CentOS Linux
```
Comment (shahmishal) [inline platforms/Linux/RPM/Oraclelinux/8/README.md:1]: We should rename this to Oracle Linux

File: platforms/Linux/RPM/Oraclelinux/8/swiftlang.spec [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+%patch0 -p1
+
+# Fix python to python3
+ln -s /usr/bin/python3 /usr/bin/python
+
+%build
+export VERBOSE=1
+
+# Run the build
+swift/utils/build-script --preset=buildbot_linux,no_assertions,no_test install_destdir=%{_builddir} installable_package=%{_builddir}/swift-%{version}-centos8.tar.gz
```
Comment (shahmishal) [inline platforms/Linux/RPM/Oraclelinux/8/swiftlang.spec:142]: We should update the tar file name

**Comment by shahmishal** [comment]
What is the status of the PR?

File: platforms/Linux/RPM/Oraclelinux/8/README.md (hunk 1 above)
Comment (tomerd) [reply platforms/Linux/RPM/Oraclelinux/8/README.md:1]:
```suggestion
# Building Swift on Oracle Enterprise Linux
```

File: platforms/Linux/RPM/Oraclelinux/8/swiftlang.spec (hunk 2 above)
Comment (tomerd) [inline platforms/Linux/RPM/Oraclelinux/8/swiftlang.spec:142]:
```suggestion
swift/utils/build-script --preset=buildbot_linux,no_assertions,no_test install_destdir=%{_builddir} installable_package=%{_builddir}/swift-%{version}-oel8.tar.gz
```

**Comment by tomerd** [comment]
@shahmishal this was waiting for a review

**Review by shahmishal — APPROVED** [review]

**Pull request merged**